tests/Integrations/Curl/CurlSandboxedTest.php

```diff
@@ -2,7 +2,546 @@
 
 namespace DDTrace\Tests\Integrations\Curl;
 
-final class CurlSandboxedTest extends CurlIntegrationTest
+use DDTrace\Format;
```

Comment:
Of all of these integrations, they were copy/pasted from the non sandboxed version, correct? And there were no modifications to any of them? It's quite impossible to detect any tiny changes in the diffs. :)

Removing the sandboxed tests and changing the default parent value of `IS_SANDBOX` to `true` would make the integrations way easier to review - especially if there are tiny changes to them. Lol.

Reply:
There are minor modifications to some because they added or overwrite methods in the base. Not sure what to do about the reviewing pain. Leaving the original but changing IS_SANDBOX to true wouldn't be enough, because the behavior changed.